.gitignore

```diff
@@ -12,3 +12,6 @@ Cargo.lock
 
 # MSVC Windows builds of rustc generate these, which store debugging information
 *.pdb
+
+# Files generated by IC
+.dfx
```

Comment:
[minter] Please add newline in the end of the text files

Reply:
Done